### ISSUE TITLE
Add support for Assembla to openhatch/oh-bugimporters

### ISSUE THREAD

**Issue by eeshangarg** [openhatch/oh-mainline#1664]
Hello!

I am aware that there already exists an [issue] (https://github.com/openhatch/oh-bugimporters/issues/76) for this in the [openhatch/oh-bugimporters] (https://github.com/openhatch/oh-bugimporters) repository. The reason I have submitted this issue here as well is because I am interested in working on this and filing an issue here felt like a great way to get answers and feedback since there are more people "watching" this repo than the [openhatch/oh-bugimporters] (https://github.com/openhatch/oh-bugimporters) repo. However, if you think that this separate issue is not really needed, please feel free to close it.

I have a question before I start working on this. We should use the [Assembla Developer API] (http://api-doc.assembla.com/) just like we use the GitHub API for our GitHub bug importers (for getting the data in JSON, in the case of the Assembla API), right? What's the recommended approach here? It would mean a lot to me if someone could please give me some guidance so that I can start working on this soon. :-)

Thanks & regards,
Eeshan Garg

**Comment by paulproteus** [comment]
Yeah! If there is an Assembla JSON API, by all means let's use that.

**Comment by eeshangarg** [comment]
Hi @paulproteus!

I have another question. So there are a couple of ways using which we can interact with the Assembla API. And I am a bit confused as to which [authentication scheme] (http://api-doc.assembla.com/content/authentication.html) would be the most suitable for writing a bug importer, I would really appreciate it if you could please give me some advice regarding this as well. :-)

Thanks & regards,
Eeshan Garg

**Comment by ehashman** [comment]
@eeshangarg: just from briefly looking at that documentation, I believe client credentials would be the way to go.

We do need changes in both oh-bugimporters and oh-mainline to bring a new tracker online, so if you're ready to work on this, it's good that you've filed this issue. Make sure you file one against bugimporters too!
